js/cryptopia.js

```diff
@@ -849,6 +849,9 @@ module.exports = class cryptopia extends Exchange {
             let nonce = this.nonce ().toString ();
             body = this.json (query, { 'convertArraysToObjects': true });
             let hash = this.hash (this.encode (body), 'md5', 'base64');
+            if (this.secret.length % 4) {
+                this.secret += '=' * (4 - this.secret.length % 4);
```

Comment:
Unfortunately, can't merge it this way, multiplying a character by number works in Python only... so `'=' * 4` isn't transpileable...

```
mbp:ccxt igorkroitor$ node
> '=' * 4
NaN
>
```

However, I'm wondering why would you need to fix the secret? That padding can only be wrong if the user made a mistake of double-clicking the secret to word-select it (that fails to select the ending `==` signs in most of browsers). If that is the case, however, we suppose that the secret is incorrect and the user should pay attention when copying credentials. Is that the issue you've been experiencing?

Reply:
Yup that was the issue I was facing. Thanks for clearing it up!

Reply:
@lottaeouss no worries, you're welcome! Looking forward to seeing more PRs from you )